src/widget-textbox.ts

```diff
@@ -28,37 +27,41 @@ export class WidgetTextbox extends LitElement {
   h2 {
     margin: 0;
     padding: 12px;
-    /* border-top-right-radius: 12px;
-    border-top-left-radius: 12px; */
+    color: var(--re-user-h2-color, --re-text-color, #000);
   }
 
   h3 {
     margin: 0;
     padding: 12px;
+    color: var(--re-user-h3-color, --re-text-color, #000);
   }
 
   p {
     margin: 0;
     padding: 0px 12px;
+    color: var(--re-user-p-color, --re-text-color, #000);
   }
 `; 
 
   render() {
+    this.style.setProperty("--re-user-h2-color", this.inputData?.title?.color)
```

Comment:
If the color is not set by the inputData would that not be set to `undefined` then, so the eventually #000 will then be applied ?

Reply:
Yes exactly 
The according css for it looks like this:

`color: var(--re-user-h2-color, --re-text-color, #000);`